src/components/Popups/ReplyField/PopupList.scss

```diff
@@ -14,11 +14,7 @@
         box-shadow: 0 1px 2px 0 rgba(0, 0, 0, .05);
     }
 
-    .ba-PopupList-item {
```

Comment:
Do we need `padding-bottom` for the ba-Popup-content element? It looks a little odd with only one result.

Reply:
> Do we need `padding-bottom` for the ba-Popup-content element? It looks a little odd with only one result.

OMG, good catch